R/peakdet.R

```diff
@@ -1,109 +1,112 @@
-#' Find_true_occurence
+#' Helper function to find the borders of a peak in a sequence of intensities
 #'
-#' Loads a .xlsx or .csv file that contains all the information needed to detect targeted compounds.
+#' @param sign_change `logical` with positions of local minima and maxima.
+#'     Length of `sign_change` needs to be the same than the length of the
+#'     original intensity vector. Eventually add leading and trailing `FALSE`.
 #'
-#' @param signs
-#' @param vector
-#' @param point
-
-#' @return
-#' @export
+#' @param index `integer(1)` with the index of the detected peak
 #'
-
-
-find_true_occurrence <- function(signs,vector,point) {
-  left_true <- NA
-  right_true <- NA
-
-  if(point == 1){
-    point = point + 1
-  }
-
-
-
-  # Search for TRUE to the left --> if sign changes and at least doens't change at the next point
-  for (i in (point - 1):1) {
-    if (signs[i] == TRUE &&  point - i > 2) {
```

Comment:
with `point - i > 2` it is actually not considering the neighboring **2** points, in contrast to the comment above.